src/pip/_internal/operations/install/wheel.py

```diff
@@ -412,6 +412,19 @@ def _raise_for_invalid_entrypoint(specification: str) -> None:
 
 
 class PipScriptMaker(ScriptMaker):
+    # Override distlib's default script template with one that
+    # doesn't import `re` module, allowing scripts to load faster.
+    script_template = r"""# -*- coding: utf-8 -*-
+import sys
+from %(module)s import %(import_name)s
+if __name__ == '__main__':
+    if sys.argv[0].endswith('-script.pyw'):
+        sys.argv[0] = sys.argv[0][: -11]
```

Comment:
This branch is unnecessary - we never generate `-script.pyw` files, so it will never happen.